documentation/asciidoc/topics/proc_exchanging_sa_tokens.adoc

```diff
@@ -6,35 +6,119 @@ After you create service account tokens on your {openshiftshort} clusters, you
+----
+oc -n <namespace> create -f sa-token.yaml
+----
+
+. Retrieve the service account token.
 +
-[source,options="nowrap",subs=attributes+]
+[source,bash,options="nowrap",subs=attributes+]
+----
+oc -n <namespace> get secrets ispn-xsite-sa-token -o jsonpath="{.data.token}" | base64 -d
 ----
-oc sa get-token lon
++
+The command prints the token in the command line.
+Copy it to deploy it in the backup {openshiftshort} cluster(s).
+
+. Log in to the backup {openshiftshort} cluster(s).
 
-eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9...
+. Add the service account token for a backup location with the following command:
++
+[source,bash,options="nowrap",subs=attributes+]
+----
+oc -n <namespace> create secret generic <token-secret> --from-literal=token=<token>
 ----
++
+The `<token-secret>` is the name of the secret that needs to be configured in the Infinispan CR.
+
+. Repeat the preceding steps on your other {openshiftshort} clusters.
+
+
+[#expiring-tokens]
+== Using a bound service account token
+
+NOTE: The token generated by this procedure expires.
+The token needs to be refreshed periodically, otherwise, the Operator will lose access to the remote {openshiftshort} cluster.
+If you are looking for a non-expiring token, follow the procedure in <<non-expiring-tokens>>.
 
 .Procedure
 
 . Log in to an {openshiftshort} cluster.
-. Add the service account token for a backup location with the following command:
+
+. Create a bound token for the service account
 +
-[source,options="nowrap",subs=attributes+]
+[source,bash,options="nowrap",subs=attributes+]
 ----
-oc create secret generic <token-name> --from-literal=token=<token>
+oc -n <namespace> create token <service-account>
 ----
 +
-For example, log in to the {openshiftshort} cluster at **NYC** and create a `lon-token` secret as follows:
+The command prints the token in the command line.
+Copy it to deploy it in the backup {openshiftshort} cluster(s).
++
+TIP: Use the command option `--duration` to set the token expiring date.
+
+. Log in to the backup {openshiftshort} cluster(s).
+
+. Add the service account token for a backup location with the following command:
 +
-[source,options="nowrap",subs=attributes+]
+[source,bash,options="nowrap",subs=attributes+]
 ----
-oc create secret generic lon-token --from-literal=token=eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9...
+oc -n <namespace> create secret generic <token-secret> --from-literal=token=<token>
 ----
 +
+The `<token-secret>` is the name of the secret that needs to be configured in the Infinispan CR.
+
 . Repeat the preceding steps on your other {openshiftshort} clusters.
+
+.Renew expiring tokens
+
+When a token expires, a new token needs to be created.
+Log in to the backup {openshiftshort} cluster and delete the secret `<token-secret>`.
+
+[source,bash,options="nowrap",subs=attributes+]
+----
+oc -n <namespace> delete secrets <token-secret>
+----
+
+Repeat this procedure to create a new token and generate a new `<token-secret>`.
```

Comment:
Is this sufficient, or do we also have to restart the Operator pods? I remember you mentioning this step in a meeting...

Reply:
The operator took about ~1 min to fetch the updated secret but we don't need to restart it :partying_face: